docker_images/mbed-os-env/README.md

```diff
@@ -0,0 +1,48 @@
+# mbed-OS development environment docker image
+
+This docker image is the official mbed-OS development environment.
```

Comment:
This docker image is the official Mbed OS development environment.
* It is based on Ubuntu 20.04
* Python and CMake are installed
* Arm-none-eabi-gcc toolchain is installed
* Latest released version of mbed-cli and mbed-greentea are installed
* All other Mbed OS dependency tools are installed.

fixing some letters in our product names

Reply:
thanks, will update thesee.